poolqc/pooling_qc_sample.py

```diff
@@ -244,56 +237,26 @@ def update_percent_mapped_to_code(run, rows, derived_virtual_selected_rows, colo
         )
         data.append(
             go.Scatter(
-                y=list(d['library']),
-                x=list(d['Threshold']),
-                orientation='h',
-                name=inx,
+                x=list(d['library']),
+                y=list(d['Threshold']),
+                mode='markers+lines',
                 line={
                     'width': 3,
-                    'color': 'rgb(204,0,0)',
+                    'color': 'rgb(0,0,0)',
                     'dash': 'dash',
-                },
-                mode='lines'))
+                },))
 
     return {
         'data': data,
         'layout': {
-            'title': 'Per Cent Mapped to Coding (RNA) ',
-            'yaxis': {'title': 'Sample', 'automargin': True},
-            'xaxis': {'title': 'Per Cent', 'range': [0, 100]},
+            'title': 'Index Clusters per Sample. Passed Samples: %s Threshold: %s' % (Index_Pass, Index_Threshold),
+            'xaxis': {'title': 'Sample', 'automargin': True},
+            'yaxis': {'title': 'Clusters'},
             'showlegend': False,
-            'height': (len(run) * 30) + 150
+
         }
     }
 
 
-@app.callback(
-    [dep.Output('map_code', 'figure'),
-     dep.Output('SampleIndices', 'figure')],
-    [dep.Input('select_a_run', 'value'),
-     dep.Input('lane_select', 'value'),
-     dep.Input('Summary Table', 'derived_virtual_data'),
-     dep.Input('Summary Table', 'derived_virtual_selected_rows')])
-def update_all_graphs(run_alias, lane_alias, rows, derived_virtual_selected_rows):
```

Comment:
I'd prefer to keep an "update all graphs" function which wraps `update_sampleindices`, as it'll make it easier if we need to add new graphs later.